dev/.env

```diff
@@ -1 +1,7 @@
 # User provided environment variables
+consul_enterprise=true
+nomad_enterprise=true
+vault_enterprise=true
+consul_acl=false
+consul_acl_default_policy=deny
+nomad_acl=false
```

Comment:
Avoid changes in this file, its only for local run
```suggestion
```

Reply:
I pushed latest changes to test on another machine. This will be gone when testing is done